client/components/test-mode-notice/index.tsx

```diff
@@ -143,7 +143,9 @@ const getNoticeContent = (
 								'Viewing test %1$s. To view live %1s, disable test mode in {{settingsLink}}%2s settings{{/settingsLink}}.',
 								'woocommerce-payments'
 							),
-							currentPage,
+							'deposits' === currentPage
```

Comment:
We'll need to consider how & when we update the URL path / page ID. It would be great if we can ship this all in one release – I _think_ that's doable since the page ID is not referenced externally.

Reply:
Aha, the URL can be fixed as part of the nav change:

- #9525